master/internal/rm/kubernetesrm/pods.go

```diff
@@ -1221,6 +1239,25 @@ func (p *pods) cleanUpPodHandler(podHandler *pod) error {
 	return nil
 }
 
+func (p *pods) handleGetSlotsRequest(agentID string) *apiv1.GetSlotsResponse {
+	agentResp := p.handleGetAgentRequest(agentID)
+	slots := agentResp.Agent.Slots
+	response := &apiv1.GetSlotsResponse{}
+	for _, slot := range slots {
+		response.Slots = append(response.Slots, slot)
+	}
+	return response
```

Comment:
style nit: https://github.com/uber-go/guide/blob/master/style.md#use-var-for-zero-value-structs, in most of the declarations in this change. to me, its nice to visually distinguish zero val structs, slices and such very clearly from non zero val. take it or leave it, not really followed elsewhere, but i follow this.

so this would be
```suggestion
	var response apiv1.GetSlotsResponse
	for _, slot := range slots {
		response.Slots = append(response.Slots, slot)
	}
	return &response
```

Reply:
but really i'd just change this entire block to
```suggestion
	return &apiv1.GetSlotsResponse{Slots: slots}
```

Reply:
`slots` is a map, but `response.Slots` is a slice!

Reply:
ah yeah, then 
```suggestion
	return &apiv1.GetSlotsResponse{Slots: maps.Values(slots)}
```